src/x11/glcanvas.cpp

```diff
@@ -124,4 +124,13 @@ int wxGLCanvas::GetColourIndex(const wxColour& col_)
     return col.GetPixel();
 }
 
+double wxGLCanvas::GetOpenGLScaleFactor() const
```

Comment:
This should really be in `src/gtk/glcanvas.cpp` and not here...

Reply:
Yes, of course, thanks for noticing this (well, both you and Travis :-/). Will fix.